src/common/locations.ts

```diff
@@ -3,7 +3,24 @@ import US_STATE_DATASET from 'components/MapSelectors/datasets/us_states_dataset
 import { each, sortBy, takeRight } from 'lodash';
 import { assert } from './utils';
 
-// TODO(michael): Add types, move more common code here.
+// TODO(michael): Move more common code here.
```

Comment:
The core change in this file is just adding the helpers for getting state / county stuff (location name, location URL) by FIPS, but I also added some types while I was in here.